src/libbson/fuzz/CMakeLists.txt

```diff
@@ -5,8 +5,10 @@ set_property(TARGET fuzz_test_properties APPEND PROPERTY INTERFACE_LINK_OPTIONS
 add_executable(fuzz_test_init_from_json EXCLUDE_FROM_ALL fuzz_test_init_from_json.c)
 target_link_libraries(fuzz_test_init_from_json PRIVATE fuzz_test_properties)
 
-add_executable(fuzz_test_validate EXCLUDE_FROM_ALL fuzz_test_validate.c)
+add_executable(fuzz_test_validate EXCLUDE_FROM_ALL fuzz_test_validate.cpp)
```

Comment:
Recommend updating the description of the `MONGO_FUZZ` CMake option to include "(requires C++) to document the additional toolchain dependency required to build fuzz testing targets. Also consider guarding these targets with `if (CMAKE_CXX_COMPILER)` as done for the `mongoc-cxx-check` target.